doc/nrf/releases/release-notes-2.3.0.rst

```diff
@@ -0,0 +1,1020 @@
+  * New :ref:`wifi_sr_coex_sample` sample.
+
+* Improved:
+
+  * The :ref:`matter_lock_sample` sample has been extended to support switching between Matter over Thread and Matter over Wi-Fi during application operation.
+  * The :ref:`wifi_shell_sample` sample by adding the Power Save feature.
+
+See :ref:`ncs_release_notes_230_changelog` for the complete list of changes.
+
+Sign up for the `nRF Connect SDK v2.3.0 webinar`_ to learn more about the new features.
+
+The official nRF Connect for VS Code extension has also been improved.
+See the `latest release notes for nRF Connect for Visual Studio Code`_ for more information.
+
+Release tag
+***********
+
+The release tag for the |NCS| manifest repository (|ncs_repo|) is **v2.3.0**.
+Check the :file:`west.yml` file for the corresponding tags in the project repositories.
+
+To use this release, check out the tag in the manifest repository and run ``west update``.
+See :ref:`cloning_the_repositories` and :ref:`gs_updating_repos_examples` for more information.
+
+For information on the included repositories and revisions, see `Repositories and revisions for v2.3.0`_.
+
+IDE and tool support
+********************
+
+`nRF Connect extension for Visual Studio Code <nRF Connect for Visual Studio Code_>`_ is the only officially supported IDE for |NCS| v2.3.0.
+SEGGER Embedded Studio Nordic Edition is no longer tested or expected to work with |NCS| v2.3.0.
+
+:ref:`Toolchain Manager <gs_app_tcm>`, used to install the |NCS| automatically from `nRF Connect for Desktop`_, is available for Windows, Linux, and macOS.
+
+Supported modem firmware
+************************
+
+See `Modem firmware compatibility matrix`_ for an overview of which modem firmware versions have been tested with this version of the |NCS|.
+
+Use the latest version of the nRF Programmer app of `nRF Connect for Desktop`_ to update the modem firmware.
+See :ref:`nrf9160_gs_updating_fw_modem` for instructions.
+
+Modem-related libraries and versions
+====================================
+
+.. list-table:: Modem-related libraries and versions
+   :widths: 15 10
+   :header-rows: 1
+
+   * - Library name
+     - Version information
+   * - Modem library
+     - `Changelog <Modem library changelog for v2.3.0_>`_
+   * - LwM2M carrier library
+     - `Changelog <LwM2M carrier library changelog for v2.3.0_>`_
+
+Known issues
+************
+
+Known issues are only tracked for the latest official release.
+See `known issues for nRF Connect SDK v2.3.0`_ for the list of issues valid for the latest release.
+
+.. _ncs_release_notes_230_changelog:
+
+Changelog
+*********
+
+The following sections provide detailed lists of changes by component.
+
+IDE and tool support
+====================
+
+* Removed the :file:`scripts/tool-version-minimum.txt` file because only one toolchain version is used for testing.
+  The table in :ref:`gs_recommended_versions` has been updated accordingly to no longer specify minimum tool versions.
+
+MCUboot
```

Comment:
Shouldn't this be part of the MCUboot section further down?

Reply:
I think it's a subtle difference - this is not a change coming from MCUboot itself but our Kconfig option that defines how we use MCUboot.